### DISCUSSION TITLE
How will tea magics handle name collisions?

### DISCUSSION THREAD

**Discussion by ddobrinskiy** [pkgxdev/pkgx#404] [Ideas]
For example, these two crates are both callable with the `btm` command, while being unrelated to each other

- https://crates.io/crates/bottom
- https://crates.io/crates/btm

Once they are both packaged with tea, how should tea magic handle this?

cc @mxcl @mfts

**Comment by mxcl** [comment]
I believe:

1. If one is already installed, use that
2. Otherwise prompt the user to choose, then next time *1.* triggers

Possibly we could have a ranking system based on the protocol page rank longer term.
